packages/frameworks/vue/src/checkbox/checkbox.props.ts

```diff
@@ -5,6 +5,7 @@ import { declareEmits } from '../utils'
 export const props = {
   checked: {
     type: String as PropType<Context['checked']>,
```

Comment:
The `checked` type can be the same as `defaultChecked` according to the same context. Are there any typing errors there?

Reply:
Oh yes, I missed it. Because we used type assertion `PropType<Context['checked']>`, so there is no typing error.